aspnetcore/security/authorization/razor-pages-authorization.md

```diff
@@ -21,9 +21,9 @@ The sample app uses [cookie authentication without ASP.NET Core Identity](xref:s
 
 ## Require authorization to access a page
 
-Use the <xref:Microsoft.Extensions.DependencyInjection.PageConventionCollectionExtensions.AuthorizePage*> convention via <xref:Microsoft.Extensions.DependencyInjection.MvcRazorPagesMvcBuilderExtensions.AddRazorPagesOptions*> to add an <xref:Microsoft.AspNetCore.Mvc.Authorization.AuthorizeFilter> to the page at the specified path:
+Use the <xref:Microsoft.Extensions.DependencyInjection.PageConventionCollectionExtensions.AuthorizePage*> convention to add an <xref:Microsoft.AspNetCore.Mvc.Authorization.AuthorizeFilter> to the page at the specified path:
```

Comment:
I dropped this _via AddRazorPagesOptions_ thing in general, as it woudl need to be changed and I don't think it added any value.